src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "AvenirBook";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.form1 {
+  margin-top: 20px;
+}
+
+.bar {
+  width: 100%;
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.largebar {
+  height: 70px;
+  padding-left: 62px;
+
+}
+
+.largebar::placeholder {
+  font-family: "AvenirBook", sans-serif;
```

Comment:
move to html selector and remove repeats in other classes, instead this inherit it